mne/tests/test_cov.py

```diff
@@ -496,6 +497,17 @@ def test_cov_scaling():
     _undo_scaling_array(data, picks_list, scalings=scalings)
     assert_allclose(data, evoked.data, atol=1e-20)
 
+    # check that scaling is undone
```

Comment:
If I understand it properly here we test that there are no collateral effects. That `_regularized_covariance` input data remain unchanged (that's the only assert). And this is a regression test linked to this PR.

I would change the comment

```
# check that input data remain unchanged. gh-5698
```